packages/algob/src/builtin-tasks/sign-multisig.ts

```diff
@@ -0,0 +1,59 @@
+import algosdk from "algosdk";
+import fs from "fs-extra";
+
+import { task } from "../internal/core/config/config-env";
+import { loadRawSignedTxnFromFile, writeToFile } from "../lib/files";
+import { signMultiSig } from "../lib/msig";
+import { RuntimeEnv } from "../types";
+import { TASK_SIGN_MULTISIG } from "./task-names";
+
+export interface TaskArgs {
+  file: string
+  accountName: string
+  out: string
+}
+
+async function multiSignTx (
+  taskArgs: TaskArgs,
+  runtimeEnv: RuntimeEnv
+): Promise<void> {
+  const signerAccount = runtimeEnv.network.config.accounts.find(acc => acc.name === taskArgs.accountName);
+  if (signerAccount === undefined) {
+    console.error("No account with the name \"%s\" exists in the config file.", taskArgs.accountName);
+    return;
+  }
+  const txFile = loadRawSignedTxnFromFile(taskArgs.file);
+  const tx = algosdk.decodeObj(txFile);
```

Comment:
Didn't try it.

Reply:
@ratik21 `decodeSignedTransaction` accepts the blog part which we receive as an output from this function( `decodeObj`).
Therefore, there is no alternative to using this function, because while exporting transaction I used `encodeObj`.